lib/streamlit/testing/v1/element_tree.py

```diff
@@ -553,6 +568,23 @@ def __init__(self, proto: HeadingProto, root: ElementTree):
         super().__init__(proto, root, "title")
 
 
+@dataclass(repr=False)
+class Json(Element):
+    proto: JsonProto = field(repr=False)
+
+    expanded: bool
```

Comment:
nit: We don't need to set `expanded` in the constructor since this will be directly resolved via the proto by the overwritten `__getattr__`, or?  (This seems to have changed since my last review)